nautobot/core/templates/components/panel/body_content_objects_table.html

```diff
@@ -1,4 +1,7 @@
 {% load render_table from django_tables2 %}
-{% with table=body_content_table %}
-    {% render_table table 'inc/table.html' %}
-{% endwith %}
+<form id="{{form_id}}" method="post">
```

Comment:
Probably need an `if form_id` conditional here so that we don't add an `id="None"` in the default case?